servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpApiConversions.java

```diff
@@ -290,24 +290,36 @@ public static BlockingStreamingHttpClient toBlockingStreamingClient(StreamingHtt
      * @return {@link ServiceAdapterHolder} containing the service adapted to the streaming programming model.
      * @deprecated Use overload with {@link HttpExecutionStrategy} rather than {@link HttpExecutionStrategyInfluencer}
```

Comment:
While you are here, can you please update this description with a link to the correct method?